runtime/vm/LayoutFFITypeHelpers.hpp

```diff
@@ -75,7 +76,47 @@ class LayoutFFITypeHelpers
 	}
 
 	/**
```

Comment:
Please correct the signatures of functions with parameters of type `char **`: those parameters should be of type `const char **`; i.e.
```
	getIntFromLayout(const char **layout)
	getStructFFITypeElements(const char **layout)
	getStructFFIType(const char **layout)
	getStructFFITypeElements(const char **layout)
	getStructFFIType(const char **layout)
	getArrayFFIType(const char **layout, UDATA nElements)
	parseStruct(const char **currentStructSig, bool *isAllSP, bool *isAllDP, U_8 *first16ByteComposTypes, UDATA *currentIndex)
	setByteCellforStruct(const char **currentStructSig, bool *isAllSP, bool *isAllDP, U_8 *first16ByteComposTypes, UDATA *currentIndex, UDATA arrayLength)
```
The documentation for each should note that those parameters are `[in/out]` and explain what updates should be expected (Where should the caller's pointer be upon return?).

Other functions that receive `char *` pointers should be updated to expect `const char *`
```c
	encodeUpcallSignature(const char *cSignature, J9UpcallSigType *sigType)
	encodeUpcallPrimitive(const char *cSignature)
	encodeOuterStruct(const char *structSig, U_32 sizeInByte)
```

Reply:
Fixed as suggested above.